python-sdk/src/astro/options.py

```diff
@@ -15,6 +15,15 @@ def to_dict(self) -> dict:
         return attr.asdict(self)
 
 
+def check_required_option(load_options: Optional[LoadOptions], option_name: str) -> bool:
```

Comment:
```suggestion
def is_required_option(load_options: Optional[LoadOptions], option_name: str) -> bool:
```

Reply:
With this method, we are checking if the required option is present or not.`is_required_option` to me gives an impression to check if a given option is required. I think it should be `check_required_option` WDYT?

Reply:
Hm, I see. What about `contains_required_option`? It would be more explicit!

Reply:
Yup, this sounds much better. Updated it.